benchmark/benchmark.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Time taken to insert the data"
+time sqlite3 test.db < data.sql
+
+echo "Time taken to query"
+time sqlite3 test.db "select * from datasets where id=59;" >/dev/null 2>&1 \
```

Comment:
These tests are wrong, what we need is the following:
- select all datasets, e.g. select * from datasets and save all results
- select one dataset from step above and find blocks for it
  select * blocks where dataset=/a/b/c
  here /a/b/c is an example of dataset
- from step above take one block find files for it
  select * from files where block=/a/b/c#123
  again here /a/b/c#123 is an example.
- select files for a given dataset, e.g.
SELECT * FROM FILES AS F JOIN BLOCKS AS B ON F.BLOCKID=B.ID JOIN DATASETS AS D ON F.DATASETID = D.ID WHERE d.dataset=/a/b/c/

The point is that we will not know the ids, but code will operate either with dataset or block names.